asciidoc/courses/app-python/modules/0-setup/lessons/1-setup/lesson.adoc

```diff
@@ -112,7 +112,7 @@ To start the project, run the following command:
 .Start the project
 [source,sh]
 ----
-flask run
+FLASK_APP=api flask run
```

Comment:
?

Reply:
Looks like this made it in from the python course somehow, I will remove it